eng/test-configuration.json

```diff
@@ -3,6 +3,7 @@
   "defaultOnFailure": "fail",
   "localRerunCount": 2,
   "retryOnRules": [
-    { "testAssembly": { "wildcard": "System.Net.*" } }
+    { "testAssembly": { "wildcard": "System.Net.*" } },
+    { "failureMessage": { "wildcard": "Timed out after * waiting for the browser to be ready" } }
```

Comment:
do you know if this does a substring match? e.g. the message will normally have `messagePrefix` and `psi.FileName` before/after, do we need to add `*` at the beginning/end too?

Reply:
https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.Helix/Sdk#wildcard-value-with--wildcard suggests, IIUC, that it should match a substring.

Reply:
Looks like that didn't work - https://dev.azure.com/dnceng-public/public/_build/results?buildId=86544&view=ms.vss-test-web.build-test-results-tab&runId=1812800&resultId=100139&paneView=dotnet-dnceng.dnceng-build-release-tasks.helix-test-information-tab . I'll open a new PR, and explicitly reproduce the error.

Reply:
Oh, and reading the docs again:

```md
Raw string (e.g. "rule string")

True if the property value exactly matches the string

{"wildcard": "value with * wildcard"}

The same as a raw string, but "*" can match any number of characters, and "?" can match one character
```

*same as raw string*, and `raw string` says that it will match the *property value* exactly.
And the property value I'm using here:
```
failureMessage
The failure message logged by the test
```

So, you were likely correct. I will try regex instead.